### ISSUE TITLE
About Conf_mask

### ISSUE THREAD

**Issue by lilyzhuang** [andy-yun/pytorch-0.4-yolov3#37]
Hi, thank you for your efforts in this code. I was confused about the conf_mask in yolo_layer.py.

Before caculating the true conf_mask, it is inializated as conf_mask  = torch.ones (nB, nA, nH, nW). #L47
However, it's modifed to zero according to ignore_ix. #L76. And it is assigned to 1 agian based on the selected anchor box. # L99. 

Many thanks.

**Comment by andy-yun** [comment]
For clarity, I separated conf_mask as obj_mask and noobj_mask. Would you once more inspect the codes.

**Comment by lilyzhuang** [comment]
> For clarity, I separated conf_mask as obj_mask and noobj_mask. Would you once more inspect the codes.

Got it. Thank you very much. Btw, have you ever tried the different weights for different loss term?

**Comment by andy-yun** [comment]
Sure, the latest code shows the best performance on various training conditions. 
If you have tried some combinations and it shows the best performance, please let me know.
